diff --git a/src/util/config/config.cpp b/src/util/config/config.cpp
index 8d9219da..6d2600c4 100644
--- a/src/util/config/config.cpp
+++ b/src/util/config/config.cpp
@@ -1,4 +1,5 @@
 #include <array>
+#include <filesystem>
 #include <fstream>
 #include <sstream>
 #include <iostream>
@@ -699,14 +700,22 @@ namespace dxvk {
   Config Config::getUserConfig() {
     Config config;
 
-    // Load either $DXVK_CONFIG_FILE or $PWD/dxvk.conf
-    std::string filePath = env::getEnvVar("DXVK_CONFIG_FILE");
-
-    if (filePath == "")
-      filePath = "dxvk.conf";
-    
+    // Load $DXVK_CONFIG_FILE, $PWD/dxvk.conf, or ${prefix}/etc/dxvk.conf
+    std::array<std::string, 3> paths = {
+      env::getEnvVar("DXVK_CONFIG_FILE"),
+      "dxvk.conf",
+      "Z:@@PREFIX@@\\etc\\dxvk.conf"
+    };
+
+    std::filesystem::path filePath;
+    for (auto path : paths) {
+      filePath = std::filesystem::path(path);
+      if (exists(filePath))
+        break;
+    }
+ 
     // Open the file if it exists
-    std::ifstream stream(str::tows(filePath.c_str()).c_str());
+    std::ifstream stream(filePath);
 
     if (!stream)
       return config;
