diff -ur a/fetchDependencies b/fetchDependencies
--- a/fetchDependencies	2021-11-15 12:03:37.000000000 -0500
+++ b/fetchDependencies	2022-01-30 00:58:32.000000000 -0500
@@ -206,21 +206,7 @@
 # $2 repo url
 # $3 repo revision (commit SHA)
 function update_repo() {
-	echo "$1 repo: $2"
-	echo "$1 revision: $3"
-
-	if [ -d $1 -a -d $1/.git ]; then
-		cd $1
-		git fetch --all
-		git checkout --force $3
-		cd -  > /dev/null
-	else
-		rm -rf $1
-		git clone $2 $1
-		cd $1
-		git checkout $3
-		cd -  > /dev/null
-	fi
+	:
 }

 # Build a repository
@@ -333,7 +319,6 @@
 	update_repo ${REPO_NAME} ${REPO_URL} ${REPO_REV}

 	cd ${REPO_NAME}
-	./update_glslang_sources.py
 	rm -rf build
 	./build_info.py .        \
 		-i build_info.h.tmpl  \
